deps/v8/gypfiles/features.gypi

```diff
@@ -104,6 +104,14 @@
     'v8_untrusted_code_mitigations%': 'true',
 
     'v8_enable_handle_zapping%': 1,
+
+    'v8_enable_pointer_compression%': 'false',
+
+    'v8_enable_embedded_builtins%': 'false',
+
+    'v8_perf_prof_unwinding_info%': 0,
+
+    'v8_enable_fast_mksnapshot%': 0,
```

Comment:
@nodejs/v8-update do we want to turn any of these for node by default?

Reply:
Generally it makes sense to copy the settings from [BUILD.gn](https://github.com/v8/v8/blob/7.0-lkgr/BUILD.gn#L71), so
- Disable pointer compression. That's work-in-progress.
- Enable embedded builtins. I think that's the point of this PR. You save quite a lot of memory per isolate this way. But that does not work for ia32 yet.
- Disable handle zapping. That's for debugging, and should not be used in production.
- Disable untrusted code mitigations. That's to address Spectre, but that's outside Node.js' threat model.
- Disable fast mksnapshot. That's only to make mksnapshot run faster, giving a shorter turnaround during development.

Reply:
Btw untrusted code mitigations are actually disabled from `common.gypi` already.